### PR TITLE
User meeting 2017 page content

### PR DESCRIPTION
**Pull request by hflynn**
Completes the content on https://snoopycrimecop.github.io/www.openmicroscopy.org/community/events/12th-annual-users-meeting-2017.html which should now match https://www.openmicroscopy.org/site/community/minutes/meetings/12th-annual-users-meeting-2017

### CODE REVIEW

File: community/events/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
         <hr class="whitespace">
         <div class="row">
             <div class="small-12 medium-8 medium-offset-2">
-                <p class="callout primary">[ Hi Helen - please help with this wording. ] We are working to get the rest of the Previous Users Meeting Notes up. In the meantime, previous notes can be found here [insert link].</p>
+                <p class="callout primary">We are in the process of moving content to this new site at present. Details of earlier meetings are still available at [https://www.openmicroscopy.org/site/community/minutes/meetings] </p>
```
Comment (mtbc) [inline community/events/index.html]: not sure this link worked as intended?

File: community/events/12th-annual-users-meeting-2017.html [hunk 2]
```diff
@@ -68,19 +188,200 @@ <h5 class="event-time">09:30 - 10:45</h5>
...
+                        materials
+                    </li>
                 </ul>
                 <ul class="event-workshop">
                     <li class="event-workshop-title"><span class="event-workshop-room">Room 2</span> - Working with OME Files and Bio-Formats (developer-aimed session)</li>
-                    <li>Overview of OME Model and Formats, Bio-Formats and OME Files</li>
-                    <li><a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/Introduction-to-Bio-Formats/#/">Introduction to working with Bio-Formats</a></li>
-                    <li><a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/OME-Files-C++/cpp-dundee-2017.pdf">Introduction to working with OME Files</a></li>
+                    <li>
+                        Overview of OME Model and Formats, Bio-Formats and OME
```
Comment (mtbc) [inline community/events/12th-annual-users-meeting-2017.html]: indents differently when rendered

File: community/events/index.html (hunk 1 above)
Comment (hflynn) [inline community/events/index.html]: It's just a placeholder until we decide the strategy - sorry, should have probably said that in the text

File: community/events/12th-annual-users-meeting-2017.html (hunk 2 above)
Comment (hflynn) [inline community/events/12th-annual-users-meeting-2017.html]: Huh - it looks fine in Safari but I just tried Firefox instead and it does indeed look different, although it's the items underneath for this session and OME.smuggler further down the page which don't match all the rest - this line has the same indent as the list for Room 1 etc.

I'll have a play and fix that later

File: community/events/12th-annual-users-meeting-2017.html [hunk 3]
```diff
@@ -68,19 +188,140 @@ <h5 class="event-time">09:30 - 10:45</h5>
...
+                    <li class="event-workshop-title"><span class="event-workshop-room">Room 2</span> - <a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/Jupyter-Workshops/">Analyzing data in OMERO</a></li>
+                    <li>Introduction to IDR and Jupyter</li>
+                    <li>Analyzing data in OMERO using Python and R</li>
+                    <li>Working through use-cases including image 
+                        features</li>
+                </ul>
+                <ul class="event-workshop">
+                    <li class="event-workshop-title"><span class="event-workshop-room">Room 3</span> - <a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/Omero-Web/#/">OMERO.web apps</a></li>
+                    <li>Intro to OMERO.web framework</li>
+                    <li>new JSON API</li>
```
Comment (mtbc) [inline community/events/12th-annual-users-meeting-2017.html]: should have initial capital?

**Pull request merged**